pkl-cli/src/main/kotlin/org/pkl/cli/commands/RootCommand.kt

```diff
@@ -48,6 +49,11 @@ class RootCommand : NoOpCliktCommand(name = "pkl") {
       ProjectCommand(),
       DownloadPackageCommand(),
       AnalyzeCommand(),
+      CompletionCommand(
+        name = "completion",
+        help = "Generate shell completion scripts",
```

Comment:
This will be a pretty rarely used command, so, I'd prefer if this had a more specific name (in case we have something else called complete in the future).

I think `shell-completion` or `generate-completion` (the default name from Clikt) work.

Curious what @stackoverflow and @holzensp think

Reply:
Updated to `shell-completion`, which seems to be the most unambiguous.